openach/templatetags/templatehelpers.py

```diff
@@ -0,0 +1,16 @@
+from django import template
+
+register = template.Library()
+
+
+@register.simple_tag
+def relative_url(value, field_name, urlencode=None):
```

Comment:
What is the urlencode argument being passed in? It is a string with the encoded url? It's confusing b/c urlencode is the name of a method in the python standard library: https://docs.python.org/3.8/library/urllib.parse.html?highlight=urlencode#urllib.parse.urlencode